Middleware docs (#5625)
vuestorefront/vue-storefront: packages/core/docs/general/key-concepts.md

You can read more about Vue Storefront Context [here](/advanced/context)

## Middleware

> When it comes to the networking layer, Vue Storefront uses a middleware that's is a bridge between front-end and other backends (eCommerce or 3rd party services). The front-end always calls middleware that is redirecting requests to correlated destinations. It allows developers to implement a custom logic that injects into the lifecycle of the requests or even create custom API endpoints if it's needed.

Reviewer: ```suggestion
When it comes to the networking layer, Vue Storefront uses a middleware that is a bridge between front-end and other backends (eCommerce or 3rd party services). The front-end always calls middleware that is redirecting requests to correlated destinations. It allows developers to implement custom logic to inject into the lifecycle of the requests or even create custom API endpoints if needed.
```

> You can read more about Vue Storefront Middleware [here](/advanced/server-middleware)

Reviewer: ```suggestion
You can read more about Vue Storefront Middleware on the [Server Middleware](/advanced/server-middleware) page.
```

## Integrations

> Even though high-level APIs are the same for all Vue Storefront integrations they're different on the low level (data formats, search params). Check the docs of a specific platform on the left side under the "eCommerce integrations" tab to learn about them.

Reviewer: ```suggestion
Even though high-level APIs are the same for all Vue Storefront integrations, they are different on the low level (data formats, search params). Check the documentation for a specific platform to learn more.
```
